agent/recognition/binarymatch.py

```
@@ -1398,7 +1407,14 @@ def _finalize_hit(self, *, context, argv, node, key_roi, caller,
         configured_hsv = params.get("hsv_ranges", _RED_RANGES_DEFAULT)
         meta = {
             "mode": "preset" if caller else "standalone",
-            "box": list(result_box), "conf": conf, "parts": parts,
+            "box": list(result_box),
+            # box 是加过 roi 偏移的坐标，偏移量取决于本次是独立还是预设模式；
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)